config/PythiaDelphes_config.py

```diff
@@ -36,17 +50,25 @@
 ## Define Delphes card
 delphesCard="config/FCChh_DelphesCard_WithDipole_v00.tcl"
 
-## Define Delphes objects (allParticles, partons and stable particles are flushed automatically)
-delphesOutCol=[]
-#delphesoutcol=["muons","electrons","photons","jets","mets","hts"]
-
 ## Define Delphes input HepMC and optionaly (non-standard) ROOT output
 ##  - if hepMC file not defined --> data read-in from Gaudi data store (Inputs)
 ##  - if ROOT file not defined --> data written-out to Gaudi data store (Ouputs)
 delphesHepMCInFile=""
+delphesRootOutFile=""
 #delphesHepMCInFile="data/ZLEP_toAll.hepmc"
-delphesRootOutFile="DelphesOutput.root"
-#delphesRootOutFile=""
+#delphesRootOutFile="DelphesOutput.root"
+
+## Define Delphes outpu arrays for studied objects -> various Delphes modules
+## provide the same type of particle with different level of processing ...
+## Needed due to required output to FCC-EDM!
+delphesMuonsOutArray    ="MuonMomentumSmearing/muons"
+delphesElectronsOutArray="ElectronEnergySmearing/electrons"
+delphesChargedOutArray  ="ChargedHadronMomentumSmearing/chargedHadrons"
+delphesNeutralOutArray  ="Hcal/eflowNeutralHadrons"
+delphesPhotonsOutArray  ="PhotonEfficiency/photons"
+delphesJetsOutArray     ="JetEnergyScale/jets"
+delphesMETsOutArray     ="MissingET/momentum"
+delphesSHTsOutArray     ="ScalarHT/energy"
```

Comment:
Why do you define these variables if you use their value only once each? Just simplify.